testsuite/cluster-test/src/cluster_swarm/templates/libra_node_spec_template.yaml

```diff
@@ -56,7 +56,7 @@ spec:
       name: config
     env:
     - name: RUST_LOG
-      value: "warn"
```

Comment:
do we want to commit this? why isn't this info at least :O

Reply:
We should commit debug log level
It was the case before but was configured via `STRUCT_LOG_LEVEL`. Then we changed logging to only look at RUST_LOG, but did not update this spec.

Since we are here, @zekun000 do you mind also removing STRUCT_LOG_LEVEL two lines below?